frontend/src/layout/navbar.tsx

```diff
@@ -1,14 +1,19 @@
-import React from 'react';
+import React from "react";
 import { AppBar, Button } from '@material-ui/core';
 
-import { RouteComponentProps } from '@reach/router';
+import { RouteComponentProps } from "@reach/router";
 
 export const NavBar: React.FC<RouteComponentProps> = () => {
   return (
     <AppBar position="fixed">
+      <div style={{ display: "flex", justifyContent: "center" }}>
       <Button color="inherit" href="/dashboard">
         People's Dashboard
       </Button>
+      <Button color="inherit" href="/home">

```

Comment:
Let's remove any additional code not specified for the tasks. We must maintain our solution as clean as possible.